src/clr-core/modal/modal.element.scss

```diff
@@ -0,0 +1,120 @@
+@import './../styles/tokens/generated/index';
+@import './../styles/mixins/utils';
+@import './../styles/utils/close';
+@import './../styles/utils/a11y';
+
+:host {
+  --backdrop-opacity: 0.85;
+  --backdrop-color: #{$cds-token-color-neutral-900};
+  --box-shadow-color: #{$cds-token-color-action-500};
+  --border-radius: #{$cds-token-global-border-radius};
+  --border-color: #{$cds-token-color-action-600};
+  --background-color: #{$cds-token-color-neutral-0};
+  --close-icon-color: #{$cds-token-color-neutral-600};
+  --close-icon-color-hover: #{$cds-token-color-neutral-700};
+  --content-box-shadow-color: #{$cds-token-color-neutral-1000};
+  --title-color: #{$cds-token-typography-h3-color};
+  --title-font-family: #{$cds-token-typography-header-font-family};
+  --title-font-weight: #{$cds-token-typography-h3-font-weight};
+  --width: calc(8 * #{$cds-token-space-size-13});
+}
+
+:host([size='sm']) {
+  --width: calc(4 * #{$cds-token-space-size-13});
+}
+
+:host([size='lg']) {
+  --width: calc(12 * #{$cds-token-space-size-13});
+}
+
+:host([size='xl']) {
+  --width: calc(16 * #{$cds-token-space-size-13});
+}
+
+.private-host {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: 1050; // TODO: restore using map-get to get the z-index
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+button.close {
+  font-size: $cds-token-space-size-8;
+  line-height: $cds-token-space-size-8;
+
+  &:focus,
+  &:hover,
+  &:active {
+    --close-icon-color: var(--close-icon-color-hover);
+  }
+
+  cds-icon {
+    fill: var(--close-icon-color);
+    // per measurement, this results in an icon that is 16x16...
+    @include equilateral($cds-token-space-size-8);
+  }
+}
+
+.modal-dialog {
+  position: relative;
+  z-index: 1050; // TODO: restore using map-get to get the z-index
+  width: var(--width);
+  max-width: 100%;
+
+  $modal-boxshadow-y: $cds-token-space-size-1;
+  $modal-boxshadow-spread: $cds-token-space-size-2;
+
+  .modal-content {
+    background-color: var(--background-color);
+    border-radius: var(--border-radius);
+    box-shadow: 0 $modal-boxshadow-y $modal-boxshadow-spread $modal-boxshadow-spread
+      rgba(var(--content-box-shadow-color), 0.2);
+    line-height: 0; // this is so that what goes inside content doesn't have weird extra spaces..
+  }
+}
+
+.modal-title {
+  color: var(--title-color);
+  font-size: $cds-token-typography-h3-font-size;
```

Comment:
If users can't change the size of the title's font, it seems like the rest of the customizations are less relevant. Could we allow them to change the font-size here (maybe they want an h1 or h4? and then set the line-height to a sane value they can override with CSS if the unthinkable occurs and they want a 2em line-height with h4 text? 😂 

A sane line-height for header text (from Colin's recommendations) appears to be `1.1em` or thereabouts. These titles shouldn't wrap often.

Reply:
Also, why are not using the line-height erasers here?

Reply:
If I remember I think from our previous discussion we decided to treat the header title the same as layouts and dont apply styles. This will make the styles API smaller/simpler as well as give them more flexibility. Example use case they can add a heading with sub heading or easily control layout with additional actions.

```html
<cds-modal>
  <cds-modal-header>
    <h3 cds-text="title">My Modal</h3>
  </cds-modal-header>
</cds-modal>
```

I think that would be the most flexible option. If we dont want to go that way then all the title styles should exist under `cds-modal-header`.

```scss
// modal-header.element.scss
:host {
   --color: $cds-token...;
   --font-size: $cds-token...;
   --font-weight: $cds-token...;
   --line-height': $cds-token...;
   --letter-spacing': $cds-token...;
   // font family automatically inherits from the app so it does not need to be a exposed prop
}
```

Reply:
I think your suggestion makes sense to keep in line with our guidelines. I'm taking out the styling (and customizability) of the title so that the end user can define the look and feel of the header text using appropriate typography.